test/onnx/test_op_consistency.py

```diff
@@ -307,7 +307,7 @@ def test_output_match(self, device: str, dtype: torch.dtype, op):
                     self.run_test(model, inputs, rtol=rtol, atol=atol)
 
 
-for opset in onnx_test_common.TESTED_OPSETS:
+for opset in onnx_test_common.ONNXRT_SUPPORTED_OPSETS:
```

Comment:
Should we test opset18? We don’t support it in onnx.export

Reply:
We don't until we add helper functions inputs/attributes changes into symbolic_opset*.py. I think CI would break if we bump it in this PR.

Reply:
What do you mean opset 18 is not supported by the onnx exporter? [col2im op]
(https://github.com/pytorch/pytorch/blob/main/torch/onnx/symbolic_opset18.py) was implemented at `symbolic_opset18.py`, for example. This is the [onnxruntime unit test](https://github.com/pytorch/pytorch/blob/b0d109f29f650d04c501dfde72623e71442ed7d0/test/onnx/test_pytorch_onnx_no_runtime.py#L1118) for it

Maybe some tests need to be skipped for opset 18, but col2im is an op needed by several models out there, including internal customers

Reply:
(https://github.com/pytorch/pytorch/pull/107829) So opset18 support is dependent on the various Reduce* ops being updated because its axis attribute was promoted to input. There is a good number of ops using reduce directly or implicitly, and fixing all of them would not be realistic for 2.1, if at all. To avoid confusions for users we say opset18 is not supported, so they don't export a model to opset18 only to find more errors down the road. However, users can still choose to ignore the warning if they know what they are doing (e.g. when they need to use col2im)